resources/wlcg-scope-tag-migration/WLCGScopeTagList.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/*
+ * A simple script to home some variables useful for the WLCG Scope tag
+ * migration.
+ */
+
+$wlcgScopesList = ["alice", "atlas", "belle", "cms", "dune", "lhcb"];
+$tierScopesList = ["tier0", "tier1", "tier2", "tier3"];
```

Comment:
Is every tier (theoretically) possible for every VO?

Reply:
Indeed.